extensions/container-image/container-image-docker/deployment/src/main/java/io/quarkus/container/image/docker/deployment/DockerProcessor.java

```diff
@@ -141,7 +141,7 @@ protected String createContainerImage(ContainerImageConfig containerImageConfig,
         if (!useBuildx && pushContainerImage) {
             // If not using buildx, push the images
             loginToRegistryIfNeeded(containerImageConfig, containerImageInfo, executableName);
-            pushImages(containerImageInfo, executableName);
+            pushImages(containerImageInfo, executableName, dockerConfig);
```

Comment:
Why do you use as word "dockerConfig" and not "containerConfig" ?

Reply:
That's what it's already called